eng/pipelines/runtime-wasm-dbgtests.yml

```diff
@@ -0,0 +1,41 @@
+trigger: none
+
+variables:
+  - template: /eng/pipelines/common/variables.yml
+
+jobs:
+
+#
+# Evaluate paths
+#
+- template: /eng/pipelines/common/evaluate-default-paths.yml
```

Comment:
This is fine for now. But in a follow up we should change this to use these from runtime-extra-platforms-wasm.yml with a new `debuggerTestsOnly` parameter.